index.js

```diff
@@ -1,10 +1,14 @@
 window.requestIdleCallback =
     window.requestIdleCallback ||
-    function(cb) {
+    function(cb, options) {
+        var options = options || {};
+        var timeout = options.timeout || 1;
```

Comment:
This isn't actually used as the timeout.

Reply:
I don't understand

Reply:
But I think we have to implement all options in polyfill

Reply:
Isn't the point of the timeout to change the timeout argument given to setTimeout?

Reply:
The previous code always say to callback `didTimeout` is `false`, but that is not correct

Reply:
So do you any questions? or you are not sure about this PR